src/Command/Core/CoreClearCommand.php

```diff
@@ -41,11 +41,22 @@ protected function configure()
      */
     protected function execute(InputInterface $input, OutputInterface $output)
     {
-        $application = $this->getApplication();
-        /** @var \Gush\Application $application */
-        $config = $application->getConfig();
-        (new Filesystem())->remove($config->get('cache-dir'));
+        $cacheDir = $this->getConfig()->get('cache-dir');
 
-        $output->writeln('Cache has been cleared.');
+        // Confirm is to ensure the Cache-directory is what the user expects
```

Comment:
// Ensures that the cache directory is what the user expects (thought this last part is not clear)

what do you mean by what the user expects? what is the user expecting?